Support/templates/Article.tex

```diff
@@ -5,7 +5,6 @@
 % Copyright (c) `date "+%Y"` ${TM_ORGANIZATION}.
 \documentclass{article}
 
-\usepackage{polyglossia}
```

Comment:
Sorry, I did not intend to do any changes to the Article template, it was an unintentional side-effect of basing my Memoir template on it, but trying to use pdflatex instead of xelatex. Indeed, it should not be affected.